.github/workflows/tests.yml

```diff
@@ -18,10 +18,21 @@ jobs:
         with:
           dafny-version: "3.4.2"
 
-      - uses: actions/setup-python@v1
+      - name: Install Dafny report generator
+        run: dotnet tool install --global dafny-reportgenerator
 
       - name: Install lit 
         run: pip install lit OutputCheck
 
-      - name: Verify/Test Dafny code
-        run: lit --time-tests -v .
+      - name: Verify Dafny Code
+        run: lit --time-tests -v --param 'dafny_params=/verificationLogger:trx /verificationLogger:csv' .
+
+      - name: Generate Report
+        run: find . -name '*.csv' | xargs -t dafny-reportgenerator summarize-csv-results --max-duration-seconds 10
```

Comment:
```suggestion
        run: find . -name '*.csv' -print0 | xargs -0 --verbose dafny-reportgenerator summarize-csv-results --max-duration-seconds 10
```

Reply:
What happens if there are too many `.csv` reports?

Reply:
Yeah I'm a bit concerned that will be an issue in the future and cut an issue here: https://github.com/dafny-lang/dafny-reportgenerator/issues/4. The current command length is 1505 so we've got a fair bit of headroom under the 4K limit at least (if it applies to the commands that xargs generates).